<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teris_game</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/image/neko22.ico</normaloff>:/image/neko22.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>171</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>40</number>
     </property>
     <item>
      <widget class="QPushButton" name="single_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>单人游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pairbutton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>多人游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QRadioButton" name="EasyradioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::TabFocus</enum>
       </property>
       <property name="text">
        <string>Normal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="HardradioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::TabFocus</enum>
       </property>
       <property name="text">
        <string>Hard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CrazyradioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <bold>true</bold>
         <underline>true</underline>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::TabFocus</enum>
       </property>
       <property name="text">
        <string>Crazy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionhelp">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
   <property name="text">
    <string>how to play</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
